plugins/infinispan100/src/main/java/org/radargun/service/Infinispan100ServerService.java

```diff
@@ -0,0 +1,163 @@
+package org.radargun.service;
+
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import org.radargun.Service;
+import org.radargun.config.Property;
+import org.radargun.traits.Clustered;
+import org.radargun.traits.ProvidesTrait;
+import org.radargun.utils.Utils;
+
+import static org.radargun.service.ServerLogPattern.*;
+
+@Service(doc = InfinispanServerService.SERVICE_DESCRIPTION)
+public class Infinispan100ServerService extends Infinispan80ServerService {
+
+   @Property(doc = "Default server port for Hotrod, Rest and other operations. If this property is set a non-WildFly based server will be used.")
+   private String defaultServerPort;
```

Comment:
Just curious, if we have a boolean value `use-serverng` and all other fields with the default values, could we reuse all our benchmark files and set the value in the trigger job?

Reply:
We don't have `use-serverng` property anymore